src/components/DelegationPool/DelegationList/index.tsx

```diff
@@ -171,52 +178,29 @@ const DelegationLists: React.FC = () => {
   ];
   return (
     <>
-      <TopSearchContainer>
-        <SearchContainer ref={tableRef}>
-          <StyledInput
-            placeholder={t("common.searchPools")}
-            onChange={(e) => setValue(e.target.value)}
-            value={value}
-            onKeyUp={(e) => {
-              if (e.key === "Enter") {
-                setSearch(value);
-                history.replace({ search: stringify({ ...pageInfo, page: 1 }) });
-              }
-            }}
-          />
-          <SubmitButton
-            onClick={() => {
-              setSearch(value);
-              history.push(routers.DELEGATION_POOLS, {
-                tickerNameSearch: (value || "").toLocaleLowerCase()
-              });
-              history.replace({ search: stringify({ ...pageInfo, page: 1 }) });
-            }}
-          >
-            <CustomIcon
-              data-testid="search-icon"
-              icon={HeaderSearchIconComponent}
-              fill={theme.palette.secondary[0]}
-              stroke={theme.palette.secondary.light}
-              height={22}
-              width={22}
+      <TopSearchContainer sx={{ justifyContent: "end" }}>
+        <Box display="flex" gap="10px">
+          <ShowRetiredPools>
+            {t("glassary.showRetiredPools")}
+            <AntSwitch
+              checked={isShowRetired}
+              onChange={(e) => {
+                setIsRetired(e.target.checked);
+                history.replace({ search: stringify({ ...pageInfo, page: 0, retired: e.target.checked }) });
+              }}
             />
-          </SubmitButton>
-        </SearchContainer>
-        <ShowRetiredPools>
-          {t("glassary.showRetiredPools")}
-          <AntSwitch
-            checked={isShowRetired}
-            onChange={(e) => {
-              setIsRetired(e.target.checked);
-              history.replace({ search: stringify({ ...pageInfo, page: 0, retired: e.target.checked }) });
-            }}
-          />
-        </ShowRetiredPools>
+          </ShowRetiredPools>
+          <CustomFilterMultiRange />
+        </Box>
       </TopSearchContainer>
       <Table
         {...fetchData}
-        columns={columns}
+        columns={columns.filter((col) => {
```

Comment:
Can we write more elegant way like this. but please check after changing if all works?
{columns.filter(col => !(["governanceParticipationRate", "votingPower"].includes(col.key) && !IS_CONWAY_ERA))}